doc/API-Mastodon.md

```diff
@@ -1,7 +1,7 @@
 # Mastodon API
 
 * [Home](help)
-  * [Using the APIs](help/api)
+  * [Using the APIs](https://docs.joinmastodon.org/api/)
```

Comment:
This link is meant to be the link to the Friendica API documentation, not the Mastodon API.

Reply:
Yes, sorry.